src/gluonts/model/deepar/_estimator.py

```diff
@@ -164,6 +170,25 @@ def __init__(
     def create_transformation(self) -> Transformation:
         return Chain(
             [
+                RemoveFields(
```

Comment:
For the block `
                   (
                        []
                        if self.use_feat_dynamic_real
                        else [FieldName.FEAT_DYNAMIC_REAL]
                    )
`

This alternative might be more readable:

```
remove_field_names = [FieldName.FEAT_DYNAMIC_CAT, FieldName.FEAT_STATIC_REAL]
if self.use_feat_dynamic_real:
    remove_field_names.append(FieldName.FEAT_DYNAMIC_REAL)

return Chain(
    [
        RemoveFields(field_names=remove_field_names),
        ...
```

(if you disagree, just discard the comment)